photon-core/src/main/java/org/photonvision/jni/RknnDetectorJNI.java

```diff
@@ -68,9 +68,9 @@ public static class RknnObjectDetector {
 
         private static final CopyOnWriteArrayList<Long> detectors = new CopyOnWriteArrayList<>();
 
-        public RknnObjectDetector(String modelPath, List<String> labels) {
+        public RknnObjectDetector(String modelPath, List<String> labels, RknnJNI.ModelVersion version) {
             synchronized (lock) {
-                objPointer = RknnJNI.create(modelPath, labels.size());
+                objPointer = RknnJNI.create(modelPath, labels.size(), version.ordinal(), 0);
```

Comment:
The core we use should be a static number we increment and then modify by 3 each time I think?